src/public/infernode/views/Capture/CapturePage.tsx

```diff
@@ -9,80 +11,171 @@ import {
+  }, [isAppCapturing]);
+
   return (
-    <Stack direction="horizontal" gap={3}>
-      <CaptureSidebar />
-      <Stack direction="vertical" gap={3}>
-        <Card>
-          <h4>Choose Your App</h4>
-          <Form>
-            <Form.Group className="mb-3" controlId="formGroupAppName">
-              <Form.Label>App Name</Form.Label>
-              <Form.Control type="text" placeholder="Enter App Name" />
-            </Form.Group>
-            <Form.Group className="mb-3" controlId="formGroupPath">
-              <Form.Label>Relative Path of Entry Point</Form.Label>
-              <Form.Control type="text" placeholder="/enter/relative/path" />
-            </Form.Group>
-          </Form>
-          <div className="d-grid gap-2">
-            <Button variant="success" size="lg">
-              Start Application
-            </Button>
-            <Button variant="danger" size="lg">
-              Stop Application
-            </Button>
-          </div>
-        </Card>
-        <Card>
-          <h4>Predetermined Capture Length</h4>
-          <Col>
-            <Form.Group className="mb-3" controlId="formBasicEmail">
-              <Form.Label>Duration of Capture</Form.Label>
-              <Form.Control type="text" placeholder="Enter Duration" />
-              <Form.Text className="text-muted">
-                Please enter the duration you would like to capture in seconds.
-              </Form.Text>
-            </Form.Group>
-          </Col>
-          <Col>
-            <Button variant="success" size="lg">
-              Start
-            </Button>
-          </Col>
-        </Card>
-        <Card>
-          <h4>Custom Length Capture</h4>
-          <div className="d-grid gap-2">
-            <div className="row">
-              <div className="col-6">
-                <Button variant="success" size="lg">
-                  Start
-                </Button>
-              </div>
-              <div className="col-6">
-                <h6>Click this button to start your capture!</h6>
-              </div>
+    <div>
+      <Stack direction="horizontal" gap={3}>
+        <CaptureSidebar />
+        <Stack direction="vertical" gap={3}>
+          <Card>
+            <h4>Choose Your App</h4>
+            <Form>
+              <Form.Group className="mb-3" controlId="formGroupAppName">
+                <Form.Label>App Name</Form.Label>
+                <Form.Control type="text" id="appName" placeholder="Enter App Name" />
+              </Form.Group>
+              <Form.Group className="mb-3" controlId="formGroupPath">
+                <Form.Label>Relative Path of Entry Point</Form.Label>
+                <Form.Control type="text" id="relativePath" placeholder="/enter/relative/path" />
+              </Form.Group>
+            </Form>
+            <div className="d-grid gap-2">
+              <Button
+                variant="success"
+                size="lg"
+                onClick={() => {
```

Comment:
What are your thoughts on refactoring this out of an inline function into a named function declared outside of the return scope? Given that it's more than a one-liner, it makes it a bit hard to read the overall module having functionality in multiple places.

Reply:
Good suggestions! Thank you!